hub/android/wsa/index.md

```diff
@@ -226,21 +226,36 @@ The properties for the Windows Subsystem for Android VM are listed below. Hardco
 | Build.VERSION.SDK_INT | 32 |
 | Build.BOARD | windows |
 
-## Security
+## Redirect to Windows apps
+
+Windows Subsystem for Android™️ automatically redirects intents for files and common URI schemes to the corresponding Windows default file/protocol handler (if multiple intent filters match, users see a "Windows default app" option in the chooser dialog). Supported file intents include [ACTION_VIEW](https://developer.android.com/reference/android/content/Intent#ACTION_VIEW), [ACTION_EDIT](https://developer.android.com/reference/android/content/Intent#ACTION_EDIT), [ACTION_SEND](https://developer.android.com/reference/android/content/Intent#ACTION_SEND), and [ACTION_SEND_MULTIPLE](https://developer.android.com/reference/android/content/Intent#ACTION_SEND_MULTIPLE), which copy the file to the Windows Downloads folder before opening it. Supported URI intents include [ACTION_VIEW](https://developer.android.com/reference/android/content/Intent#ACTION_VIEW) for the http/https schemes and [ACTION_VIEW](https://developer.android.com/reference/android/content/Intent#ACTION_VIEW) and [ACTION_SENDTO](https://developer.android.com/reference/android/content/Intent#ACTION_SENDTO) for the mailto scheme.
+
+Android apps can also manually redirect to Windows apps using custom URI schemes. Set the intent action to `com.microsoft.windows.LAUNCH_URI` and add a string extra to the intent named `com.microsoft.windows.EXTRA_URI` with the custom URI as the value. For example, to launch the Windows Calculator app from an Android app (Java):
 
-Windows Subsystem for Android performs per-file encryption that is software-based.
+```java
+Intent intent = new Intent("com.microsoft.windows.LAUNCH_URI");
+intent.putExtra("com.microsoft.windows.EXTRA_URI", "ms-calculator:");
+ 
+try {
+    startActivity(intent);
+} catch (ActivityNotFoundException e) {
+    // Not running in Windows Subsystem for Android™️ (or running on an old build).
+}
+```
+
+## Security
 
 Both Windows kernel-mode drivers and Windows applications running at medium integrity level (IL) can inspect arbitrary Android containers and Android app memory. There are no plans to add detection for cheats/macro/bot/suspicious behaviors detection in the short-term.
 
 Developers querying `getSecurityLevel` will get `SECURITY_LEVEL_SW_SECURE_CRYPTO`. Learn more about `getSecurityLevel` in the [Android API Reference guide](https://developer.android.com/reference/android/media/MediaDrm#getSecurityLevel(byte[])).
```

Comment:
Is this still true?

Reply:
I think so.  This is a MediaDrm API, so I think it's referring to things like WideVine.  I don't think File Based Encryption is related to DRM.